lavorare-insieme/meeting.md

```diff
@@ -0,0 +1,17 @@
+# Meeting
```

Comment:
Mi viene istintivo pensare che i punti andrebbero riorganizzati in una sorta di ordine cronologico:
- le cuffie mi servono in generale <-> `Investi in un paio di cuffie decenti`
- c'è un meeting, devo partecipare? -> `Non aver paura di dire di no`
- devo organizzare un meeting, di cosa si parlerà? -> `Stabilisci un'agenda`
- `Sii Puntuale`
- `Prendi appunti`

Reply:
@matteogalliani che ne dici se la riorganizziamo così?

Prima del meeting:

- Cuffie
- Decidi se devi partecipare
- Stabilisci un'agenda

Durante il meeting:

- Sii puntuale
- Prendi appunti
- ...

Dopo il meeting:

- Manda un recap

Reply:
🔝 
mi viene in mente che abbiamo notato significativi miglioramenti nella produttività dei meeting stilando dei `what's next`, cioè una lista di cose da fare nel meeting o l'agenda del meeting successivo. Non so se si sovrappone al recap...